src/components/filter-row/filter-row.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classnames from 'classnames';
+import VisibleIcon from '../icons/visible';
+import InvisibleIcon from '../icons/invisible';
+import { VisibilityControl } from '../ui/visibility-control/visibility-control';
+import { RowText } from '../ui/row-text/row-text';
+
+import './filter-row.scss';
+
+export const FilterRow = ({
+  allUnchecked,
```

Comment:
there's some problem here, when I uncheck everything, the visibility icons disappear.

Reply:
it should be fixed now, the PR is ready for review for the design now 😄